index.js

```diff
@@ -23,6 +23,10 @@ var sockets = {}
 if (typeof chrome !== 'undefined') {
   chrome.sockets.udp.onReceive.addListener(onReceive)
   chrome.sockets.udp.onReceiveError.addListener(onReceiveError)
+} else {
+  var undefinedChromeErrorMsg = 'Undefined chrome ref, could not register on-receive listener'
+  console.error(undefinedChromeErrorMsg)
+  throw new Error(undefinedChromeErrorMsg)
```

Comment:
Logging an error *and* throwing seems a bit excessive. It looks like we just log errors throughout most of this package, so maybe we should remove the `throw`?